tekton/publish.yaml

```diff
@@ -150,11 +150,11 @@ spec:
       OUTPUT_BUCKET_RELEASE_DIR="/workspace/output/bucket/previous/$(inputs.params.versionTag)"
 
       # Publish images and create release.yaml
-      ko resolve --preserve-import-paths -t $(inputs.params.versionTag) -f /workspace/go/src/github.com/tektoncd/pipeline/config/ > $OUTPUT_BUCKET_RELEASE_DIR/release.yaml
+      ko resolve --platform=all --preserve-import-paths -t $(inputs.params.versionTag) -f /workspace/go/src/github.com/tektoncd/pipeline/config/ > $OUTPUT_BUCKET_RELEASE_DIR/release.yaml
```

Comment:
For this to work we'll need to make sure the ko image we use is updated with a recent enough ko (> version 0.6.0).

That's configured here, in the plumbing repo: https://github.com/tektoncd/plumbing/blob/master/tekton/images/ko/Dockerfile#L18

And after we make that change we'll need to rebuild and publish the image, so it can be used here.

Do you want to make the plumbing change? If not, I can to unblock you.